tests/autoyast/clone.pm

```diff
@@ -19,11 +19,27 @@
 use strict;
 use base 'console_yasttest';
 use testapi;
+use version_utils 'is_sle';
 
 sub run {
     my $self = shift;
     assert_script_run 'rm -f /root/autoinst.xml';
-    assert_script_run 'yast2 --ncurses clone_system', 400;
+    script_run('(yast2 --ncurses clone_system;echo ZYPPER-DONE) | tee /dev/$serialdev', 0);
```

Comment:
copy `ZYPPER-DONE` to here looks confused, you can echo something like `yast2-clone_system-status-\$?`